NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 24.0.4 2022-08-11
```

Comment:
This version hasn't been released yet, so it should not have a release date

```suggestion
## 24.0.4 IN-PROGRESS
```

Reply:
Text modified